### DISCUSSION TITLE
Mobile Milestone’s

### DISCUSSION THREAD

**Discussion by CuzImBisonratte** [community/community#10987] [Mobile]
I would be happy if there would be a way to view the milestones on the mobile phone. Especially if the whole thing would integrate with iOS-widgets.

**Comment by amrehman** [comment]
Thanks a lot for your feedback, and great idea about the iOS Widget! Really appreciate it.

This is a missing functionality in our apps, we are going to put this on our backlog and try to get to it as soon as possible.
Please keep the feedback coming!

**Reply by seafella** [reply]
I was trying to convince someone less technical to use github for tracking progress on tasks using ios and I couldn't believe this wasn't in the app. I was even more surprised to find some several years old requests for this feature along with unfulfilled promises just as old. Any progress here? Is this part of the app going to be prioritized?

**Comment by mhd813** [comment]
ع

**Comment by CuzImBisonratte** [comment]
Unmarked the original answer, so this Discussion doesn't seem old/resolved.
